pallets/ajuna-awesome-avatars/src/types/avatar/versions/v3.rs

```diff
@@ -208,16 +208,20 @@ impl<T: Config> ForgerV3<T> {
 		}
 
 		let (matching_indexes, match_count, mirror_count) =
-			target.dna.clone().into_iter().zip(other.dna.clone()).enumerate().fold(
+			array_1.into_iter().zip(array_2).enumerate().fold(
```

Comment:
why do we change name here?

Reply:
Because we want to compare only the progress_arrays part of the dna, not the whole dna